### ISSUE TITLE
trkleene seems to require an argument, contrary to docs

### ISSUE THREAD

**Issue by SKalt** [kaby76/Domemtech.Trash#25]
During `trparse /tmp/gram.g4 |  trkleene`, I get a nonzero exit with 
```
ERROR(S):
  A sequence value not bound to option name is defined with few items than
  required.

  --help          Display this help screen.
  --version       Display version information.
  value pos. 0
```
This appears to be a problem with [the definition of the command-line interface](https://github.com/commandlineparser/commandline/blob/master/src/CommandLine/Text/SentenceBuilder.cs#L141)

**Comment by kaby76** [comment]
Sorry, I think when I wrote the tool, I didn't know how to best do the arg parsing. The tool should take an optional arg: if there isn't an XPath arg point to a rule, then it should just go through all the rules. If there is an XPath string passed, it should point anywhere in the rule because it's hard to remember "oh yes, we need to point to a specific terminal in the rule". I fix this up today, and make a release.

**Comment by kaby76** [comment]
The fixed will be in v0.11.0 of the trkleene.

**Comment by SKalt** [comment]
I've verified that version v0.11.0 of `trkleene` works; thank you for the quick fix!
